fastlane/download_metadata.swift

```diff
@@ -57,6 +57,22 @@ let languages = [
     "zh-Hant": "zh-tw",
 ]
 
+func shouldUpdateSubtitle(locale: String) -> Bool {
+    return isIn20211210Experiment(locale: locale) == false
+}
+
+func shouldUpdateKeywords(locale: String) -> Bool {
+    return isIn20211210Experiment(locale: locale) == false
+}
+
+func isIn20211210Experiment(locale: String) -> Bool {
+    // The experiment should run on every English locale. See details in P2 post ref `pcbrnV-440-p2`.
+    //
+    // We could have an allow list, but given the mostly static nature of
+    // locale codes, the following check seems enough.
+    return locale.starts(with: "en")
```

Comment:
@mokagio Shouldn't we check if this is for WordPress or Jetpack? If I am not missing something, this prevents the downloads for both Jetpack and WordPress apps whereas the experiment is only being done for WordPress.

Fwiw, I don't see the English language in Jetpack GlotPress so I guess it won't make a difference right now.